charts/opentelemetry-target-allocator/values.yaml

```diff
@@ -0,0 +1,36 @@
+################################################################################
+# Target Allocator
+#
+# The Target Allocator is running as a deployment with a replica count of 1.
+# It discovers scraping configurations from ServiceMonitor and PodMonitor CRDs and
+# assigns them to collectors.
+# Related documentation: https://github.com/open-telemetry/opentelemetry-operator/tree/main/cmd/otel-allocator
+################################################################################
+
+targetAllocator:
+  image: ghcr.io/open-telemetry/opentelemetry-operator/target-allocator:v0.123.0
+  # Secrets to attach to the respective serviceaccount to pull docker images
+  imagePullSecrets: []
+  serviceAccount:
+    # Specifies whether a ServiceAccount should be created
+    create: true
+    # The name of the ServiceAccount to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+    # Service account annotations
+    annotations: {}
+  config:
```

Comment:
should we also set an empty [] for the scrape configs to make it clear you can do that?

Reply:
done